src/TTrigger.cpp

```diff
@@ -229,7 +229,7 @@ bool TTrigger::setRegexCodeList(QStringList regexList, QList<int> propertyList)
             QString error;
             if (!mpLua->compile(code, error, QString::fromStdString(funcName))) {
                 setError(QStringLiteral("<b><font color='blue'>%1</font></b>")
-                                 .arg(tr(R"(Error: in item %1, lua condition function "%2" failed to compile, reason:%3.)").arg(QString::number(i), regexList.at(i), error)));
+                                 .arg(tr(R"(Error: in item %1, lua condition function "%2" failed to compile, reason:%3.)").arg(QString::number(i + 1), regexList.at(i), error)));
```

Comment:
Actually there is a second detail that is off in this line, which can reasonably be fixed here: we should double-quote the reason to match the other case and have that text ending as: `... reason: "%3".` with a space after the colon:"so the reason text does not butt up against the colon like at the start of this text".

Reply:
applied.